openhands/core/config/utils.py

```diff
@@ -75,6 +75,7 @@ def set_attr_from_env(sub_config: BaseModel, prefix: str = '') -> None:
             # e.g. LLM_BASE_URL
             env_var_name = (prefix + field_name).upper()
 
+            cast_value: Any
```

Comment:
I'm not a huge fan of using "Any" to avoid type constraints, but I guess it's not a regression, so we could go with it.